src/common/low_precision_transformations/src/fake_quantize.cpp

```diff
@@ -167,10 +167,7 @@ std::shared_ptr<opset1::FakeQuantize> FakeQuantizeTransformation::fuseElementwis
         inputHighConst_f32 = fq::updateShape(fold<opset1::Subtract>(inputHighConst_f32, value), fakeQuantize->get_output_partial_shape(0));
     } else if (ov::is_type<opset1::Convert>(eltwise)) {
```

Comment:
It seems we don't need in this else-if-branch anymore.

Reply:
done, thanks